File: hw/segarra/u2/HW18ZooInPythonAndJavaScript/ec.edu.zooinjavascript.model/Bird.js
import Animal from "./Animal.js";

export default class Bird extends Animal{
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate){
        if (new.target === Bird) {
            throw new TypeError("Cannot construct Bird instances directly");
        }
        super(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate)
    }
}

File: hw/travez/u2/HW18ZooInPython/ec_edu_espe_zoo_model/Bird.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from Animal import Animal

# Ismael Travez, The Java Bandits, DCCO-ESPE

class Bird(Animal):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)
